terminus/tests/point_test.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import math
+from geometry.point import Point
+
```

Comment:
Looks like `roll` and `pitch` tests are missing (side note: try to avoid defining methods that you don't need; otherwise you will need to maintain a codebase that is not being used)